.github/workflows/update-schema.yaml

```diff
@@ -37,17 +41,17 @@ jobs:
       - name: Install widdershins
         run: npm install -g widdershins
 
-      - name: Generate new schema.yml
-        run: python manage.py spectacular --file ./chord_metadata_service/mohpackets/docs/schema.yml --validate --fail-on-warn
+      - name: Generate new schema.json
+        run: python manage.py export_openapi_schema --api chord_metadata_service.mohpackets.apis.core.api | python -m json.tool > chord_metadata_service/mohpackets/docs/schema.json
 
       - name: Convert schema to OpenAPI documentation
         run: |
-          npx widdershins ./chord_metadata_service/mohpackets/docs/schema.yml -o ./chord_metadata_service/mohpackets/docs/openapi.md -u ./chord_metadata_service/mohpackets/docs/widdershins/templates/openapi3 -c true --omitHeader true
+          npx widdershins ./chord_metadata_service/mohpackets/docs/schema.json -o ./chord_metadata_service/mohpackets/docs/openapi.md -u ./chord_metadata_service/mohpackets/docs/widdershins/templates/openapi3 -c true --omitHeader true
```

Comment:
Could you add a step to do something like https://stackoverflow.com/questions/38781929/how-to-convert-json-to-yaml-in-javascript, to minimize downstream changes?